src/lib/opts.rs

```diff
@@ -79,7 +78,7 @@ For support please contact: care@singulargenomics.com
 #[derive(Parser, Debug, Clone)]
 #[clap(name = TOOL_NAME, version = built_info::VERSION.as_str(), about=SHORT_USAGE, long_about=LONG_USAGE, term_width=0)]
 pub struct Opts {
-    /// Path to the input FASTQs.
+    /// Path to the input FASTQs or path prefix.
```

Comment:
Maybe mention how you tell the two apart?  Do you assume any path that's not an extant file is a prefix?  Or is it based on file names/extensions.